### ISSUE TITLE
Kicking ghosts with bone boots doesn't work

### ISSUE THREAD

**Issue by YesLaching** [elunna/hackem#422]
The kick passes harmlessly through.
Using uncursed +0 bone orcish boots as a dwarvish monk.
Is that just not implemented or is that a bug?

version 1.2.0-0 Beta Sun July 30 2023

**Comment by elunna** [comment]
Yep that's a little bug that got past. Thanks! It will be fixed in the next release.
